zuul-discovery/src/main/java/com/netflix/zuul/resolver/ResolverListener.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package com.netflix.zuul.resolver;
+
+import java.util.List;
+
+/**
+ * @author Argha C
+ * @since 2/25/21
+ *
+ * Listener for resolver updates.
+ */
+public interface ResolverListener<T> {
+
```

Comment:
public interfaces need to have docs for their method.

Reply:
Done.